src/Test/EventListenerIntrospectionTrait.php

```diff
@@ -9,7 +9,7 @@
 
 namespace Zend\EventManager\Test;
 
-use PHPUnit_Framework_Assert as Assert;
+use PHPUnit\Framework\Assert;
```

Comment:
@froschdesign In a trait it's only possible to define an abstract method to define that a specific method needs to be available but that does not work with `__call()` so it gets marked red in *good* IDEs. Using the direct assert seems to be more clear and the trait itself can be tested much better.